docs/howto/NewSlim.md

```diff
@@ -0,0 +1,45 @@
+**NOTE** This documentation is incomplete, for now you may be better consulting the [GO Editor Docs](http://wiki.geneontology.org/index.php/Ontology_Editing_Guide)
+
+### Adding a new Subset (Slim)
+
+See [Daily Workflow](http://ontology-development-kit.readthedocs.io/en/latest/index.html#daily-workflow) for creating branches and basic Protégé instructions. 
```

Comment:
@matentzn - this lives in ODK docs still, do we want to fully remove dependence on it?

Reply:
Yes fully.